integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -287,7 +287,7 @@ var _ = Describe("(Integration) [non-eksctl cluster & nodegroup support]", func(
 	})
 
 	It("supports deleting clusters", func() {
-		if !params.SkipDelete {
+		if params.SkipDelete {
```

Comment:
| SkipCreate       | SkipDelete           | skip this test  |
| ------------- |:-------------:| -----:|
| false      | false | false |
| false     | true      |   true |
| true |  false   |  false/true?   | 
| true |  true   |  true  |

If I have `SkipCreate` do I have a cluster to delete?

Reply:
Yes because you provided a cluster

Reply:
i'd say we don't delete it then? since it's "managed" outside of the tests?

Reply:
but then `SkipCreate` is really `SkipCreateAndDelete`

Reply:
yeah basically or seen another way, the default of `SkipDelete` actually _depends_ on the value of `SkipCreate`

Reply:
the only other test using this functionality is `crud` and that follows the `SkipCreate` only means skip creating, not deleting too. I think this is correct for now.

Really this whole thing should get refactored, but thats a different PR

Reply:
ok!